config/routes.rb

```diff
@@ -7,7 +7,8 @@
   end
   resources :users, controller: :users, only: :create
   resources :teams do
-    resources :rounds, only: [:create, :show]
+    resources :rounds, only: [:create, :index]
   end
+  resources :rounds
```

Comment:
We may want to restrict this (`only`/`except`) to not generate `:create` and `:new`, but I wouldn't be too concerned about that.